docs/rpm_packages.md

```diff
@@ -3,3 +3,27 @@
 MicroShift RPM development packages are intended to be used internally at Red Hat to facilitate the development and QE processes, **never** to be distributed to customers. The packages are produced every time a corresponding OpenShift nightly build is successful.
 
```

Comment:
If we're touching this file, let's also update the information about the following packages published in the mirror.openshift.com site:
- https://mirror.openshift.com/pub/openshift-v4/${arch}/microshift/ocp-dev-preview/ - MicroShift EC packages published for each Engineering Candidate build
-